bridge_ui/src/hooks/useGetSourceParsedTokenAccounts.ts

```diff
@@ -523,6 +527,29 @@ const createNativeCeloParsedTokenAccount = (
         });
 };
 
+const createNativeNeonParsedTokenAccount = (
+  provider: Provider,
+  signerAddress: string | undefined
+) => {
+  return !(provider && signerAddress)
+    ? Promise.reject()
+    : provider.getBalance(signerAddress).then((balanceInWei) => {
+        const balanceInEth = ethers.utils.formatEther(balanceInWei);
+        return createParsedTokenAccount(
+          signerAddress, //public key
+          WNEON_ADDRESS, //Mint key, On the other side this will be wneon, so this is hopefully a white lie.
+          balanceInWei.toString(), //amount, in wei
+          WNEON_DECIMALS,
+          parseFloat(balanceInEth), //This loses precision, but is a limitation of the current datamodel. This field is essentially deprecated
+          balanceInEth.toString(), //This is the actual display field, which has full precision.
+          "NEON", //A white lie for display purposes
+          "NEON", //A white lie for display purposes
+          neonIcon,
+          true //isNativeAsset
+        );
```

Comment:
Thinking outloud here: I wonder if we couldn't use function currying to make `createParsedTokenAccount` a bit more pleasant.